src/web/components/PortalHeader/PortalHeader.spec.tsx

```diff
@@ -17,4 +43,35 @@ describe('Portal Header tests', () => {
     const button = screen.getByRole('button');
     expect(button).toHaveTextContent('Not logged in');
   });
+
+  describe('Audit Trail in Portal Header tests based on role', () => {
+    let participantContextValue: ParticipantWithSetter;
+    let userContextValue: UserContextWithSetter;
+    let mockParticipant: ParticipantDTO;
+    let mockUser: UserWithIsUid2Support;
+
+    beforeAll(() => {
+      mockParticipant = createMockParticipant();
+      participantContextValue = createParticipantContextValue(mockParticipant);
+      mockUser = createMockUser([mockParticipant] as Participant[]);
+      userContextValue = createUserContextValue(mockUser);
+    });
+
+    test('when user has uid2support role', async () => {
+      mockUser.isUid2Support = true;
+      renderPortalHeaderWithContext(userContextValue, participantContextValue);
+      const dropdownMenuButton = screen.getByRole('button');
+      await userEvent.click(dropdownMenuButton);
+      const auditTrailMenuItem = screen.getByRole('menuitem', { name: 'Audit Trail' });
+      expect(auditTrailMenuItem).toBeInTheDocument();
+    });
+
+    test('when user does not have uid2support role', async () => {
+      mockUser.isUid2Support = false;
+      renderPortalHeaderWithContext(userContextValue, participantContextValue);
+      const button = screen.getByRole('button');
```

Comment:
nit: I think this should have the same variable name as in the previous test, for consistency, since they are referring to the same button.